app/src/main/java/com/sunasterisk/thooi/ui/notification/notifications/NotificationsFragment.kt

```diff
@@ -0,0 +1,18 @@
+package com.sunasterisk.thooi.ui.notification.notifications
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sunasterisk.thooi.R
+
+class NotificationsFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_notifications, container, false)
+    }
+}
```

Comment:
có thể viết extension function r dùng 
`fun ViewGroup.inflate(@LayoutRes layoutResource: Int) = LayoutInflater.from(context).inflate(layoutResource, this, false)`

Reply:
mấy cái fragment này là em mới gen ra thôi ạ, còn sau dùng sẽ implement BaseFragment